cloudinit/stages.py

```diff
@@ -1101,11 +1102,14 @@ def should_run_on_boot_event():
                 log_details=False,  # May have wifi passwords in net cfg
                 log_deprecations=True,
             )
-        # ensure all physical devices in config are present
-        self.distro.networking.wait_for_physdevs(netcfg)
 
-        # apply renames from config
-        self._apply_netcfg_names(netcfg)
+        if not isinstance(self.distro, Ubuntu):
```

Comment:
We should also check renderer here. While using non-netplan might not be officially supported by Ubuntu, I can foresee other renderers being used and this possibly breaking existing deployments.